core/src/components/slider/slider.ts

```diff
@@ -650,6 +657,36 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 					);
 				}
 			},
+			touchStart(event: TouchEvent, handleId: number) {
+				const handleDrag = (e: TouchEvent) => {
```

Comment:
As discussed with @ValentinNelu, it is needed to add `e.preventDefault()` here to prevent the scroll of the page from happening at the same time. It is good also to add `event.preventDefault()` at the beginning of `touchStart`. Note that both are correctly done for `mouseDown`.

Reply:
agree, forgot to add it back when testing it's necessity (y)